velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -23,6 +23,21 @@
 
 using namespace facebook::velox::exec::test;
 using facebook::velox::VectorFuzzer;
+
+template <>
+struct fmt::formatter<facebook::velox::Date> {
```

Comment:
If we want to add support for Date type to fmt::format, this code should go somewhere else where it will be available to anyone using the type, e.g. Date.h. That change should be in its own PR with a test.

Reply:
Thanks for the suggestion, added formatter changes in PR #2457, will revert the Date.h changes containing the custom formatter after #2457 is checked in.